l10n_pt_account/account_config.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
```

Comment:
Add copyright and license. There is a short form in the template at oca/maintainer-tools